Source/Core/Duality/Launcher/DualityLauncher.cs

```diff
@@ -1,22 +1,24 @@
 ﻿using System;
 using System.IO;
-using System.Linq;
 using Duality.Backend;
 using Duality.Resources;
 
 namespace Duality.Launcher
 {
-	public class DualityLauncher
+	public class DualityLauncher : IDisposable
```

Comment:
We missed that in the previous PR, but can you add XML docs for all public members?